desktop/app.js

```diff
@@ -92,6 +93,8 @@ module.exports = function main() {
     Menu.setApplicationMenu(appMenu);
 
     ipcMain.on('appStateUpdate', function (event, args) {
+      const settings = args['settings'] || {};
```

Comment:
any particular reason for the index convention here vs. `args.settings` or a one-liner…

```js
isAuthenticated = 'settings' in args && 'accountName' in args.settings;
```

Reply:
No reason other than consistency: https://github.com/Automattic/simplenote-electron/blob/develop/desktop/menus/index.js#L12